internal/handlers/handlers.go

```diff
@@ -1 +1,78 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+const folderName = "strings"
+
+func GetPage(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		http.Error(w, "method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	http.ServeFile(w, r, filepath.Join("..", "index.html"))
```

Comment:
проект надо запускать из корня, тогда не будет выхода на уровень выше
```suggestion
	http.ServeFile(w, r, "index.html")
```